pkg/apis/fleet/v1alpha1/types.go

```diff
@@ -283,10 +283,14 @@ type BackupStorage struct {
 	// Location specifies where the backup data will be stored.
 	Location BackupStorageLocation `json:"location"`
 
-	// Credentials represents the reference to the secret for the object store.
-	// The secret must contain the fields `accessKeyID` and `secretAccessKey` to access the backup storage location.
+	// SecretName represents the name of the secret containing the object store credentials.
+	// To access the backup storage location, the secret must include the following keys:
+	//
+	// - `s3-access-key`: The access key ID for S3 authentication.
```

Comment:
access-key?

Reply:
it is the name from minio "A MinIO user consists of a unique access key (username) and corresponding secret key (password)."(see https://min.io/docs/minio/kubernetes/upstream/administration/identity-access-management/minio-user-management.html#minio-idp-service-account), or we can just use `username` and `password`